src/views/ProfileView/ProfileView.js

```diff
@@ -20,7 +21,7 @@ const ProfileView = () => {
       <div className='feed-container'>
         <UserInteractions />
         <h3 className='feed-title'>Posts</h3>
-        <Feed profileIsVisible={profileIsVisible}/>
+        <Feed profileIsVisible={profileIsVisible} searchResults={searchResults} />
```

Comment:
I assume this fixed the error on the profile page? Seems so simple. Gosh, I was looking at this code for far too long.

Reply:
Yeah that seemed to do the trick haha.